Example/NotificationService/NotificationService.m

```diff
@@ -4,10 +4,10 @@
 
 @implementation NotificationService
 
-- (void)didReceiveNotificationRequest:(UNNotificationRequest *)request withContentHandler:(void (^)(UNNotificationContent * _Nonnull))contentHandler { 
+- (void)didReceiveNotificationRequest:(UNNotificationRequest *)request withContentHandler:(void (^)(UNNotificationContent * _Nonnull))contentHandler {
     
-    self.mediaUrlKey = @"myMediaUrlKey";
-    self.mediaTypeKey = @"myMediaTypeKey";
+//    self.mediaUrlKey = @"myMediaUrlKey";
```

Comment:
@nishant-clevertap why are these 2 lines commented and not removed?

Reply:
@darshanclevertap With these changes our Objc sample app was not working since here we are updating the key strings and we use default key like ct_mediaType. So I have just commented out making our app to receive same default key string and also indicating user can update the keys from here if needed.

Reply:
OK